src/main/java/pl/tomaszdziurko/jvm_bloggers/UTMParametrsConstants.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+/**
+ * UTM (Urchin Traffic Monitor) URL query parameter identifiers.
+ * 
+ * @author cslysy <jakub.sprega@gmail.com>
+ */
+public class UTMParametrsConstants {
```

Comment:
I'm wondering if this class is really necessary. These constants are used in one place only ([`BlogPostForMailItem`](https://github.com/tdziurko/jvm-bloggers/pull/100/files#diff-49fb0ede844df7fca10aefed6a422d79R77)). Maybe we should keep it there? It already contains other _UTM_ related constants (like [`UTM_SOURCE`](https://github.com/tdziurko/jvm-bloggers/pull/100/files#diff-49fb0ede844df7fca10aefed6a422d79R77) and [`UTM_MEDIUM`](https://github.com/tdziurko/jvm-bloggers/pull/100/files#diff-49fb0ede844df7fca10aefed6a422d79R77)). What do you think?